lib/init.go

```diff
@@ -5,7 +5,32 @@ import "regexp"
 func init() {
```

Comment:
Do the new regexes below add any significant startup time to the command? With so many regexes compiled at startup, wanna make sure the command still runs super fast without overheads, as these regexes would be getting compiled for every single invocation of the CLI no matter what command.

Reply:
```
$ time ipinfo grepip list.txt 
...
real	0m0.010s
user	0m0.000s
sys	0m0.011s
```

```
$ go build
$ time ./ipinfo grepip list.txt 
...
real	0m0.023s
user	0m0.009shas
sys	0m0.023s
```
execution time has increased now, difference in practice doesn't appear to be as significant, I guess

Reply:
Went up 13ms, so should be fine for now. We  can optimize if it gets worse